README.md

```diff
@@ -78,11 +102,9 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link](https://maths-magician.pages.dev/)
```

Comment:
- [ ] Well done for deploying your app 👏. However, like the previous reviewer stated, please update your live demo link with the working one as the current one shows a 404 page when clicked 👍